src/elements/BreadcrumbsDropdown/breadcrumbsDropdown.scss

```
@@ -0,0 +1,37 @@
+@import '../../scss/colors';
+@import '../../scss/shadows';
+
+.breadcrumbs__dropdown {
+  position: absolute;
+  top: 30px;
+  left: 80px;
+  z-index: 5;
+  min-width: 130px;
+  padding: 12px 0;
+  font-size: 15px;
+  line-height: 18px;
+  background: $white;
+  box-shadow: $dropdownShadow;
+
+  &-item {
+    display: block;
+    padding: 7px 16px;
+
```

After adding `data-ellipsis` let's add some `max-width`:
```suggestion
    max-width: 300px;
    
```